docs/running-on-kubernetes.md

```diff
@@ -590,41 +590,41 @@ Some of these include:
 
 See the [configuration page](configuration.html) for information on Spark configurations.  The following configurations are specific to Spark on Kubernetes.
 
-#### Spark Properties
+#### Spark Properties (`spark.kubernetes.` prefix is omitted.)
 
 <table class="table table-striped">
 <thead><tr><th>Property Name</th><th>Default</th><th>Meaning</th><th>Since Version</th></tr></thead>
 <tr>
-  <td><code>spark.kubernetes.context</code></td>
```

Comment:
This might break the doc search?

Reply:
That's true. Let me split `kubernetes` part from this PR, @yaooqinn .